AddDocs.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const DiscoveryV1 = require('ibm-watson/discovery/v1');
+const { IamAuthenticator } = require('ibm-watson/auth');
+
+const discovery = new DiscoveryV1({
+  version: '2019-04-30',
+  authenticator: new IamAuthenticator({
+    apikey: 'CMfoTLs3yHLDTVBqCWnhbJntlj3io-Ojg4QdEdGfuKQ6',
+  }),
+  url: 'https://api.us-south.discovery.watson.cloud.ibm.com/instances/3b5bd372-ed9b-4bbb-bce9-a93526329807',
```

Comment:
create an `env` file and add credentials there, here's one way to do it: https://github.com/watson-developer-cloud/node-sdk#credentials-file-easier

Reply:
thanks Mamoon I have made these changes